test/integration/good/code-gen/cpp.expected

```diff
@@ -8229,61 +8229,60 @@ class mother_model final : public model_base_crtp<mother_model> {
       local_scalar_t__ p_upper;
       p_upper = DUMMY_VAR__; 
       
-      p_upper = in__.read<local_scalar_t__>(p_real);
+      p_upper = in__.read<local_scalar_t__>();
       out__.write_free_lb(p_real, p_upper);
       local_scalar_t__ p_lower;
       p_lower = DUMMY_VAR__; 
       
-      p_lower = in__.read<local_scalar_t__>(p_upper);
+      p_lower = in__.read<local_scalar_t__>();
       out__.write_free_ub(p_upper, p_lower);
       std::vector<local_scalar_t__> offset_multiplier;
       offset_multiplier = std::vector<local_scalar_t__>(5, DUMMY_VAR__); 
       
-      assign(offset_multiplier,
-        in__.read<std::vector<local_scalar_t__>>(1, 2, 5),
+      assign(offset_multiplier, in__.read<std::vector<local_scalar_t__>>(),
```

Comment:
For this we still need to have the `1, 2, 5` dims

Reply:
d'oh, I was looking too hard at the example